src/main/java/com/olehedza/reviews/model/User.java

```diff
@@ -0,0 +1,35 @@
+package com.olehedza.reviews.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+
+@Entity
+@Table(name = "users")
+@Data
+@NoArgsConstructor
+public class User {
+    @Id
+    private String userId;
+    private String profileName;
+    @ManyToMany
+    @JoinTable(
+            name = "users_products",
+            joinColumns = @JoinColumn(name = "userId"),
+            inverseJoinColumns = @JoinColumn(name = "productId")
+    )
+    private List<Product> products = new ArrayList<>();
+    @OneToMany
+    @Cascade(value = CascadeType.SAVE_UPDATE)
```

Comment:
can we use cascade attribute of @OneToMany annotation?

Reply:
Yes, we can. Changed